tests/unmanaged_models/models.py

```diff
@@ -142,3 +142,24 @@ class Meta:
 # table *will* be created (unless given a custom `through` as for C02 above).
 class Managed1(models.Model):
     mm = models.ManyToManyField(Unmanaged1)
+
+
+# Here is an unmanaged through model between two
+class UnmanagedIntermediate(models.Model):
+    d01 = models.ForeignKey('unmanaged_models.D01', models.CASCADE, db_column='d01_id')
+    d02 = models.ForeignKey('unmanaged_models,D02', models.CASCADE, db_column='d02_id')
+
+    class Meta:
+        managed = False
+        db_table = 'unmanaged_between_two_managed'
+
+
+class D01(models.Model):
+    f_a = models.CharField(max_length=10, db_index=True)
+    f_b = models.IntegerField()
+
+
+class D02(models.Model):
+    f_a = models.CharField(max_length=10, db_index=True)
+    f_b = models.IntegerField()
+    d01 = models.ManyToManyField(D01, through=UnmanagedIntermediate, related_name='d02')
```

Comment:
Also choose a more significant name for the field, `related_name` seems unnecessary.